source/_integrations/tplink_omada.markdown

```diff
@@ -42,14 +43,15 @@ Controller versions 5.0.0 and later are supported.
 ### Network Switches
 
 - Support for enabling/disabling Power over Ethernet on a per-port basis.
+- Firmware Update notification and installation.
 
 ### Access Points
 
-- Not currently supported.
+- Firmware Update notification and installation.
```

Comment:
It is not a notification? It is an update entity, right?

Notifications are something one could create with those, but that is not something we provide?